packages/gestalt/src/Video.js

```diff
@@ -499,6 +520,7 @@ export default class Video extends React.PureComponent<Props, State> {
         <ColorSchemeProvider id="Video" colorScheme="light">
           <video
             autoPlay={playing}
+            crossOrigin="anonymous"
```

Comment:
why is this change necessary?

Reply:
the track needs to match the video origin -- if it doesn't, you need to pass `anonymous`, otherwise the browser will block the captions track from being loaded.